api/src/main/java/uk/gov/hmcts/fees/register/api/configuration/SpringSecurityConfiguration.java

```diff
@@ -10,23 +10,31 @@
 import org.springframework.security.config.annotation.web.builders.WebSecurity;
 import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
 import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+import org.springframework.security.oauth2.server.resource.web.BearerTokenAuthenticationFilter;
 import uk.gov.hmcts.reform.auth.checker.core.RequestAuthorizer;
 import uk.gov.hmcts.reform.auth.checker.core.user.User;
 import uk.gov.hmcts.reform.auth.checker.spring.useronly.AuthCheckerUserOnlyFilter;
+import uk.gov.hmcts.reform.authorisation.filters.ServiceAuthFilter;
+import uk.gov.hmcts.reform.authorisation.validators.AuthTokenValidator;
+
+import java.util.ArrayList;
+import java.util.List;
 
 import static org.springframework.security.config.http.SessionCreationPolicy.STATELESS;
 
 @Configuration
 @EnableWebSecurity
 public class SpringSecurityConfiguration extends WebSecurityConfigurerAdapter {
 
-    private AuthCheckerUserOnlyFilter authCheckerFilter;
+
+    private ServiceAuthFilter serviceAuthFilter;
+    private List<String> authorisedServices = new ArrayList<>();
 
     @Autowired
-    public SpringSecurityConfiguration(RequestAuthorizer<User> userRequestAuthorizer,
-                                 AuthenticationManager authenticationManager) {
-        authCheckerFilter = new AuthCheckerUserOnlyFilter(userRequestAuthorizer);
-        authCheckerFilter.setAuthenticationManager(authenticationManager);
+    public SpringSecurityConfiguration(final AuthTokenValidator authTokenValidator) {
+        authorisedServices.add("ccpay-bubble");
```

Comment:
Hardcoding values are fine for testing locally but you need to remove this before merging